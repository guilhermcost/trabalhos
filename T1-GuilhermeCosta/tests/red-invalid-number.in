9a1  // 9 e a1
21.95
499

