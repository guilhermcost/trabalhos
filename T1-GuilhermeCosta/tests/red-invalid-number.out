(1,NUM,"9")
(1,ID,"a1")
(2,NUM,"21")
(2,ERROR,".")
(2,NUM,"95")
(3,NUM,"499")
